tox.ini

```diff
@@ -106,24 +106,24 @@ commands    =
                      {envlogdir}/coverage.xml
 
 [covcombine]
-description = Generate combined coverage reports with py27-test coverage merged
+description = Generate combined coverage reports with py27-cov coverage merged
 commands =
-    tox -e py27-test
+    tox -e py27-cov
     sh -c 'export COVERAGE_FILE=$COVERAGE_FILE-combined; \
-    coverage combine --keep {envlogdir}/../../py27-test/log/.coverage {envlogdir}/.coverage;\
-    coverage xml  -o {envlogdir}/coverage.xml;\
+    coverage combine --keep {envlogdir}/../../py27-cov/log/.coverage {envlogdir}/.coverage;\
+    coverage xml  -o {envlogdir}/coverage-combined.xml;\
```

Comment:
If this is combining .coverage data from py2 and py3 runs, there may be compatibility issues.  IIRC there are differences in the way the 2 versions count coverage (some lines being counted in one version and not in the other).  I remember it is advised not to consume data produced by one major python release with the other.
What I deduced from that is that we should likely use xml output as a source for merging but I think I did not find a tool able to do that, and for other projects finally lost interest in this as py2 got eventually phased out - I lived with 2 separate coverage reports until py2 support got nuked.